pkg/config/config.go

```diff
@@ -421,6 +421,14 @@ type EngineConfig struct {
 	// Indicates whether the application should be running in Remote mode
 	Remote bool `toml:"remote,omitempty"`
 
+	// Number of times to retry pulling/pushing images in case of failure
+	Retry uint `toml:"retry,omitempty"`
+
+	// Delay between retries in case pulling/pushing image fails
+	// If set, container engines will retry at the set interval,
+	// otherwise they delay 2 seconds and then exponentially back off.
+	RetryDelay string `toml:"retry_delay,omitempty"`
```

Comment:
Since these are new values it's worth asking: `Retry` is a pretty global term. Would there ever be a case where we'd want to have another setting for retrying something else in a different part of the stack? If so we'd want to make these values more specific maybe.